sdk/core/azure-core-test/src/test/java/com/azure/core/test/TestProxyTests.java

```diff
@@ -195,5 +225,73 @@ public void testPlaybackWithRedaction() {
 
         assertEquals(200, response.getStatusCode());
     }
+
+    private RecordedTestProxyData readDataFromFile() {
+        File recordFile = new File(LOCAL_FILE_PATH + testInfo.getTestClass().get().getSimpleName() + "." + testInfo.getDisplayName().substring(0, testInfo.getDisplayName().length() - 2) + ".json");
```

Comment:
Instead of hardcoding the fille path you can use this:

```suggestion
        File recordFile = new File(InterceptorManager.getRecordFolder() + testInfo.getTestClass().get().getSimpleName() + "." + testInfo.getDisplayName().substring(0, testInfo.getDisplayName().length() - 2) + ".json");
```